<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>687</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>671</width>
      <height>401</height>
     </rect>
    </property>
    <widget class="WeightModePage" name="WeightModePage"/>
    <widget class="InjectionPlanPage" name="InjectionPlanPage"/>
    <widget class="InjectionRecordPage" name="InjectionRecordPage"/>
    <widget class="ManualModePage" name="ManualModePage"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>687</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="ActionWeight"/>
   <addaction name="ActionManual"/>
   <addaction name="ActionPlan"/>
   <addaction name="ActionRecord"/>
  </widget>
  <action name="ActionManual">
   <property name="text">
    <string>手动模式</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="ActionRecord">
   <property name="text">
    <string>注射记录</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="ActionPlan">
   <property name="text">
    <string>注射方案</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="ActionWeight">
   <property name="text">
    <string>体重模式</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>InjectionPlanPage</class>
   <extends>QWidget</extends>
   <header>injectionplanpage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>InjectionRecordPage</class>
   <extends>QWidget</extends>
   <header>injectionrecordpage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ManualModePage</class>
   <extends>QWidget</extends>
   <header>manualmodepage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WeightModePage</class>
   <extends>QWidget</extends>
   <header>weightmodepage.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
